### PR TITLE
basic-page-structure

### PR DESCRIPTION
**Pull request by farouqdaif1**
In this pull request several project requirements were accomplished:

Gitflow was used;
- Project is set up with webpack;
- HTML markup was written with basic styling.
- CSS file was imported into the index.js file.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            </div>
+            <div class="scores">
+                <p>Name: 100</p>
+                <p>Name: 20</p>
+                <p>Name: 50</p>
+                <p>Name: 78</p>
+                <p>Name: 125</p>
+                <p>Name: 77</p>
+                <p>Name: 42</p>
+            </div>
```
Comment (mikethreels) [inline src/index.html]: - [ ] good job making your project look like the wireframe 👍 however according to the requirements, you should use HTML just to make the layout. To view the items inside the list you are expected to use ES6 modules, which includes the usage of arrow functions, import and export and proper usage of const and let keywords. as mentioned in [the assignment](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md)

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Farouq, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi @farouqdaif1 🥇 , 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (farouqdaif1) [reply src/index.html]: Thank you

**Pull request merged**